File: app/src/main/java/com/yuyin/demo/FileItem.java
package com.yuyin.demo;

import java.io.File;

public class FileItem {
    private final File file_path;
    private String file_name = "";

    public FileItem(File path) {

        this.file_name = path.getName();
        this.file_path = path;
    }

    public String getFile_name() {
        return this.file_name;
    }

    public void setFile_name(String file_name) {
        this.file_name = file_name;
    }

    public File getFile_path() {
        return file_path;
    }
}

File: app/src/main/java/com/yuyin/demo/view/file/FileItem.kt
package com.yuyin.demo.view.file

import com.yuyin.demo.utils.YuYinUtil
import java.io.File

class FileItem(path: File) {
    var file_name = ""
    private var resultFiles: YuYinUtil.ResultFiles
    var textFile: File
    var jsonFile: File

    init {
        file_name = path.nameWithoutExtension
        resultFiles = YuYinUtil.ResultFiles.getTextAndJson(path)
        textFile = resultFiles.textFile
        jsonFile = resultFiles.json
    }
}